Sources/SwiftKafka/KafkaConsumer.swift

```diff
@@ -128,7 +128,7 @@ public final class KafkaConsumer: Sendable, Service {
     /// State of the `KafkaConsumer`.
     private let stateMachine: NIOLockedValueBox<StateMachine>
 
-    /// `AsyncSequence` that returns all ``KafkaConsumerMessage`` objects that the consumer receives.
+    /// An asynchronous sequence containing messages from the Kafka cluster.
     public let messages: KafkaConsumerMessages
```

Comment:
Shouldn't it be `KafkaConsumerEvents` including messages as a part of those events?

Reply:
For now we keep it separate because we expect the the messages to be the default case and rarely people want to use the other events.